src/utils/constants/city.ts

```diff
@@ -1,4 +1,4 @@
-export const CITY_GROUP = [
+export const SIDO_GROUP = [
   { cityName: '서울', cityNumber: 0 },
```

Comment:
```suggestion
  { sidoName: '서울', cityNumber: 0 },
```
저도 변경해놓긴 했는데 sido로 변경해주세용